gradle.properties

```diff
@@ -45,8 +45,10 @@ org.gradle.parallel=true
 
 # kotlin
 kotlin.native.ignoreDisabledTargets=true
+kotlin.native.enableKlibsCrossCompilation=true
 kotlin.mpp.stability.nowarn=true
 kotlin.mpp.enableCInteropCommonization=true
+kotlin.incremental.wasm=true
 kotlin.incremental.js.ir=false
 kotlin.incremental.js.klib=false
 kotlin.incremental.multiplatform=false
```

Comment:
Just curios, why do we disable incremental things here?

Reply:
I'm not sure. @whyoleg could you tell me if you know?

Reply:
TBH, I'm not sure either, but AFAIK those (js/multiplatform) are enabled by default in latest KGP versions. only `kotlin.incremental.wasm` and `kotlin.incremental.native` are experimental and so disabled by default (=`false`) - not sure how "stable" they are

Reply:
I think we could try to enable them, including WASM. And disable incremental WASM in case of troubles.